src/controllers/files/shared_file/_handler.py

```diff
@@ -0,0 +1,29 @@
+from src.config.soap_client import soap_client
+
+
+def shared_files_handler(token):
+    try:
+        request_data = {"token": token}
+        response = soap_client.service.share_list(request_data)
+
+        if response["error"] is True:
+            return {"msg": response["msg"]}, response["code"]
+
+        sharedFile = [
+            {
+                "name": file.name,
+                "size": file.size,
+                "isFile": file.isFile,
+                "uuid": file.uuid,
+                "ownerusername": file.ownerusername,
```

Comment:
Probably you'll need to remove this field. @SilviaPabon told me that they were thinking about removing that field since it's not completely necessary. 

Don't worry about it now; you can keep it in this PR, but keep it in mind.